package.json

```diff
@@ -105,7 +105,7 @@
     "webpack-dev-server": "3.1.14",
     "webpack-manifest-plugin": "2.0.4",
     "webpack-merge": "4.1.4",
-    "ws": "1.1.5"
+    "ws": "^7.0.0"
```

Comment:
please use the exact version, without `^`

Reply:
Okay